.all-contributorsrc

```diff
@@ -35,6 +37,24 @@
         "question",
         "doc"
       ]
+    },
```

Comment:
@tkh44 and @dralletje you all cool with [this](https://github.com/kentcdodds/babel-macros/blob/36bbc50f955e1219732e986969f0c7470774d55a/README.md#contributors)?